src/player.cpp

```diff
@@ -1122,16 +1097,14 @@ void player::update_bodytemp()
         // CONVECTION HEAT SOURCES (generates body heat, helps fight frostbite)
         // Bark : lowers blister count to -10; harder to get blisters
         int blister_count = ( has_bark ? -10 : 0 ); // If the counter is high, your skin starts to burn
-        for( const auto &intensity_dist : fires ) {
-            const int intensity = intensity_dist.first;
-            const int distance = intensity_dist.second;
-            if( frostbite_timer[bp] > 0 ) {
-                frostbite_timer[bp] -= std::max( 0, intensity - distance / 2 );
-            }
-            const int heat_here = intensity * intensity / distance;
-            temp_conv[bp] += 300 * heat_here;
-            blister_count += heat_here;
+        
+        const int h_radiation = g->get_heat_radiation( pos(), false );
```

Comment:
If `game::get_heat_radiation` is made into a free function change call to `get_heat_radiation( g->m, pos(), false );`